Suggest scienta/doctrine-json-functions for complex JSON queries. (#64)
dunglas/doctrine-json-odm: README.md

Yes.

> **Can I use native [MySQL](https://dev.mysql.com/doc/refman/en/json.html) / [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) JSON functions?**

Reviewer: ```suggestion
**Can I use the native [PostgreSQL](http://www.postgresql.org/docs/current/static/datatype-json.html) and [MySQL](https://dev.mysql.com/doc/refman/en/json.html) /JSON functions?**
```

> 

Reviewer: ```suggestion
```

> You can execute complex queries using [native queries](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/native-sql.html). 

Reviewer: ```suggestion
Yes! You can execute complex queries using [native queries](https://www.doctrine-project.org/projects/doctrine-orm/en/current/reference/native-sql.html). 
```

> To run JSON functions inside query builders or DQL statements, have a look at [scienta/doctrine-json-functions](https://github.com/ScientaNL/DoctrineJsonFunctions)

Reviewer: ```suggestion
Alternatively, install [scienta/doctrine-json-functions](https://github.com/ScientaNL/DoctrineJsonFunctions) to be able to use run JSON functions in DQL and query builders.
```
